SI/Sem.04/Pract.04/Task11.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+
+int min(int a, int b) {
+	return (a < b) ? a : b;
+}
+int min(int a, int b, int c, int d) {
+	return min(min(a, b), min(c, d));
+}
+int gcd(int a, int b) {
+	int highest = 1;
+	for (int i = 2; i <= min(a,b); i++) {
+		if (a % i == 0 && b % i == 0) {
+			highest = i;
+		}
+	}
+	return highest;
+}
+int gcd(int a, int b, int c, int d) {
+	int highest = 1;
+	for (int i = 2; i <= min(a, b, c, d); i++) {
+		if (a % i == 0 && b % i == 0 && c % i == 0 && d % i == 0) {
```

Comment:
Find gcd of the first two, then of third and fourth, then gcd of the results as you did in min of four numbers